remote/remote.go

```diff
@@ -64,7 +64,7 @@ import (
 	cv "github.com/purpleidea/mgmt/converger"
 	"github.com/purpleidea/mgmt/util"
 	"github.com/purpleidea/mgmt/util/semaphore"
-	"github.com/purpleidea/mgmt/yamlgraph"
+	yamlgraph "github.com/purpleidea/mgmt/yamlgraph2"
```

Comment:
Nack for now. Let's play with the v2 code first, sorry and leave this one-liner out for now.